### DISCUSSION TITLE
What prior work or similar projects exist to the GSF's Impact Framework?

### DISCUSSION THREAD

**Discussion by mrchrisadams** [Green-Software-Foundation/discussions#100] [Ask Anything]
Hi folks,

I'm aware of the GSF's Impact Framework, and I think I understand the goals of the project, but I'd find it really helpful to understand what prior work, or other projects might be comparable.

https://github.com/Green-Software-Foundation/if

If you know of any, would you mind adding a comment to this thread?

**Comment by florent-morel** [comment]
Hi,

I heard about CodeCarbon which seems to have several concepts in common with GSF's IF.

https://codecarbon.io/
https://github.com/mlco2/codecarbon

Might be interesting to see if we can find some convergence (if not already in place).
